setup.cfg

```diff
@@ -19,7 +19,7 @@ optimize-ast=no
 # can either give multiple identifiers separated by comma (,) or put this
 # option multiple times (only on the command line, not in the configuration
 # file where it should appear only once).
-disable=C0111,C0301,C0302,C0330,E1101,R0201,R0205,R0401,R0801,R0901,R0902,R0903,R0904,R0912,R0913,R0914,R0915,R1701,R1714,W0511,W0612
+disable=C0301,C0302,C0330,E1101,R0201,R0205,R0401,R0801,R0901,R0902,R0903,R0904,R0912,R0913,R0914,R0915,R1701,R1714,W0511,W0612
 #C0111 - missing-docstring
```

Comment:
Please remove this line as well.   There is a similar line in strict.cfg that also needs to be removed with this PR